File: branches/Andean/Fantasy.Base/ServiceModel/MissingRequiredServiceException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.Properties;

namespace Fantasy.ServiceModel
{
    [Serializable]
    public class MissingRequiredServiceException : Exception 
    {
        public MissingRequiredServiceException(Type serviceType)
            : base(string.Format(Resources.MissingRequiredServiceText, serviceType))
        {

        }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Base/ServiceModel/MissingRequiredServiceException.java
package Fantasy.ServiceModel;

import Fantasy.Properties.*;

public class MissingRequiredServiceException extends RuntimeException implements Serializable
{
	public MissingRequiredServiceException(java.lang.Class serviceType)
	{
		super(String.format(Resources.getMissingRequiredServiceText(), serviceType));

	}
}